### PR TITLE
fix Twig 3.12 compatibility

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | 
| License       | MIT

### CODE REVIEW

**Review by nicolas-grekas — APPROVED** [review]
Thanks!
In 7.2, let's bump to ^3.12 instead.

File: src/Symfony/Bridge/Twig/Tests/TokenParser/FormThemeTokenParserTest.php [hunk 1]
```diff
@@ -35,6 +36,10 @@ public function testCompile($source, $expected)
         $stream = $env->tokenize($source);
         $parser = new Parser($env);
 
+        if (class_exists(FirstClassTwigCallableReady::class)) {
+            $expected->setNodeTag('form_theme');
```
Comment (fabpot) [inline src/Symfony/Bridge/Twig/Tests/TokenParser/FormThemeTokenParserTest.php:40]: It should not be needed

Comment (xabbuh) [reply src/Symfony/Bridge/Twig/Tests/TokenParser/FormThemeTokenParserTest.php:40]: Without this change the tests fail like this:

```
1) Symfony\Bridge\Twig\Tests\TokenParser\FormThemeTokenParserTest::testCompile with data set #0 ('{% form_theme form "tpl1" %}', Symfony\Bridge\Twig\Node\FormThemeNode Object (...))
Failed asserting that two objects are equal.
--- Expected
+++ Actual
@@ @@
     'nodes' => Array (...)
     'attributes' => Array (...)
     'lineno' => 1
-    'tag' => null
+    'tag' => 'form_theme'
     'sourceContext' => Twig\Source Object (...)
     'nodeNameDeprecations' => Array ()
     'attributeNameDeprecations' => Array ()
 )
```

Comment (fabpot) [reply src/Symfony/Bridge/Twig/Tests/TokenParser/FormThemeTokenParserTest.php:40]: Sorry, I was confused, you're changing what we're expecting here, which is correct.

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @xabbuh.

**Pull request merged**